presto-product-tests/conf/product-tests-config-hdp3.sh

```diff
@@ -0,0 +1,3 @@
+HADOOP_BASE_IMAGE="prestodev/hdp3.1-hive"
+DOCKER_IMAGES_VERSION="16"
```

Comment:
it would be nice to use same version as for hdp2.6, don't we release them all at once?

Reply:
We do.